client/src/Components/VideoCard.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+
+
+function VideoCard(props) {
+
+  const {video, handleDelete} = props
+  const [rating, setRating] = useState(video.rating);
+
+  function like() {
+    setRating(rating + 1);
+  }
+
+  function dislike() {
+    setRating(rating - 1);
+  }
+
+    return (
+      <div>
+        
+        <div></div>
+        <div key={video.id}>
```

Comment:
The key prop should be applied to the outermost element that is part of a list of elements. In your case, it should be applied to the outer `<div>` that wraps each video card, not the inner div. 
FYI this is important for React to efficiently update the component list.